plugin.json

```diff
@@ -118,6 +118,13 @@
                 "type": "bool",
                 "help_text": "Allow the plugin to log the webhook event. The log level needs to be set to DEBUG.",
                 "default": false
+            },
+            {
+                "key": "ShowAuthorInCommitNotification",
+                "display_name": "Show Author in commit notification:",
+                "type": "bool",
+                "help_text": "In `Pushed commits` event notification, show commit author instead of commit committer.",
```

Comment:
Should we use the exact name of the event here, i.e. `Pushes`?

Reply:
Done, PTAL. If you want more changes (eg. in the display name or key please let me know).